scripts/content-types/lib/utils.ts

```diff
@@ -0,0 +1,71 @@
+// @ts-ignore
```

Comment:
Do we still need ignore here?

Reply:
Yes because the read package doesn't have types available